nicegui/elements/mixins/value_element.py

```diff
@@ -32,9 +32,10 @@ def handle_change(e: GenericEventArguments) -> None:
             self._send_update_on_value_change = True
         self.on(f'update:{self.VALUE_PROP}', handle_change, [None], throttle=throttle)
 
-    def on_value_change(self, callback: Callable[..., Any]) -> None:
-        """Set a callback to be invoked when the value changes."""
+    def on_value_change(self, callback: Callable[..., Any]) -> Self:
+        """Add a callback to be invoked when the value changes."""
```

Comment:
I've changed your phrasing here to be "Add", since I believe that's a little more accurate than "Set".